tests/tests.py

```diff
@@ -276,6 +276,34 @@ def test_bottom(self):
         )
 
 
+class OrderWithRespectToReorderTests(TestCase):
+    def setUp(self):
+        q1 = Question.objects.create()
+        self.u0 = TestUser.objects.create()
+        self.u1 = TestUser.objects.create()
+        self.u0_a1 = q1.answers.create(user=self.u0)
+        self.u0_a2 = q1.answers.create(user=self.u0)
+        self.u0_a3 = q1.answers.create(user=self.u0)
+        self.u1_a1 = q1.answers.create(user=self.u1)
+        self.u1_a2 = q1.answers.create(user=self.u1)
+        self.u1_a3 = q1.answers.create(user=self.u1)
+
+    def test_reorder_when_field_value_changed(self):
+        self.u0_a2.user = self.u1
+        self.u0_a2.save()
+        self.assertSequenceEqual(
+            Answer.objects.values_list("pk", "order"),
+            [
+                (self.u0_a1.pk, 0),
+                (self.u0_a3.pk, 2),
```

Comment:
Should this `order` be 1?

Reply:
Yes I think so, that would be the case if them item were deleted rather than moved to another group